src/caffe/layers/base_data_layer.cpp

```diff
@@ -77,7 +77,7 @@ void BasePrefetchingDataLayer<Dtype>::InternalThreadEntry() {
 #ifndef CPU_ONLY
   cudaStream_t stream;
   if (Caffe::mode() == Caffe::GPU) {
-    CUDA_CHECK(cudaStreamCreateWithFlags(&stream, cudaStreamNonBlocking));
+    CUDA_CHECK(cudaStreamCreate(&stream));
```

Comment:
It came from Simon's PR.
@thatguymike @slayton58 - guys, can you confirm that this is for good?